NachoSpigot-Server/src/main/java/dev/cobblesword/nachospigot/Nacho.java

```diff
@@ -46,23 +43,6 @@ public void registerCommands() {
         MinecraftServer.getServer().server.getCommandMap().register(knockbackCommand.getName(), "ns", knockbackCommand);
     }
 
-    public void registerPacketListener(PacketListener packetListener) {
```

Comment:
Why remove this?

Reply:
It's only used in the Nacho class so use of the function can just be replaced with `this.packetListeners.add(new AntiCrash());`

Reply:
It's used by other plugins, I even have a few actively using it

Reply:
Then I'll revert the change.